api/annotations.py

```diff
@@ -125,6 +125,9 @@ class AnnotationParser(object):
 
     @classmethod
     def parse(cls, _db, data, patron):
+        if patron.synchronize_annotations != True:
```

Comment:
This stops annotation synchronization from proceeding unless the patron has opted in.